secator/tasks/jswhois.py

```
@@ -0,0 +1,29 @@
+	input_types = [HOST]
+	output_types = [Tag]
+	item_loaders = [JSONSerializer()]
+	tags = ['domain', 'info']
+	input_flag = None
+	file_flag = None
+	install_version = 'latest'
+	install_cmd = 'go install -v github.com/jschauma/jswhois@[install_version]'
+	# install_github_handle = 'jschauma/jswhois'
+
+	@staticmethod
+	def on_item(self, item):
```

Ruff (0.14.4)

Unused static method argument: `self`

(ARG004)